src/env.h

```diff
@@ -401,7 +401,7 @@ class AsyncHooks : public MemoryRetainer {
   void grow_async_ids_stack();
 
   v8::Global<v8::Array> js_execution_async_resources_;
-  std::vector<v8::Local<v8::Object>> native_execution_async_resources_;
+  v8::LocalVector<v8::Object> native_execution_async_resources_;
```

Comment:
Note that this was and is still illegal usage of V8 Locals. They're not supposed to be living outside of the scope they were created in.

Reply:
Yeah, I was looking at that after. I think this needs to be changed to a `std::vector<v8::Global<v8::Object>>` instead but want to look into it a bit more.